lib/FusionInventory/Agent/Task/Deploy/Job.pm

```diff
@@ -19,6 +20,7 @@ sub new {
         uuid            => $params{data}->{uuid},
         requires        => $params{data}->{requires},
         checks          => $params{data}->{checks},
+        userchecks      => $params{data}->{userinteractions},
```

Comment:
This is not a typo, this shows the different points of view between agent and server sides:
- as `{userinteractions}` is what provides the server, server configuration is really exposed as a feature to configure user interactions.
- `{usercheck}` is really how I implemented the feature on the agent side, as user checks

In fact I really need to write some doc...

- [ ] Write the user interaction API documentaiton